src/components/EventSchedule/EventSchedule.js

```diff
@@ -21,20 +22,20 @@ const eventsSun = [
   { time: "2:00 PM", name: "closing ceremony" }
 ];
 
-const EventScheduleTitle = styled.h2`
-  color: #ef833f;
-  font-family: "Roboto", sans-serif;
-  font-size: 48px;
-  font-weight: 500;
-  position: relative;
-  margin-left: 15%;
-`;
+// const EventScheduleTitle = styled.h2`
```

Comment:
If this code is unused, delete

Reply:
For the unused code, I will go through the code after most parts are done and clean it up nicely. Some left overs are from works that were supposed to be continued later.